src/main/java/com/gizasystems/automationexercise/pages/HomePage.java

```diff
@@ -8,24 +8,28 @@ public class HomePage {
     // Variables
     private SHAFT.GUI.WebDriver driver;
     private String url = System.getProperty("baseUrl");
-
     // Locators
     private final By featuredItems_div = By.cssSelector("div.features_items");
     private final By recommendedItems_div = By.cssSelector("div.recommended_items");
     private final By categoryTitle_div= By.xpath("//h2[@class='title text-center']");
-
+    private final By scrollUpArrow_button = By.xpath("//i[@class='fa fa-angle-up']");
+    private final By slideHeading_div= By.xpath("(//h2)[2]");
     // Constructor
     public HomePage(SHAFT.GUI.WebDriver driver) {
         this.driver = driver;
     }
-
     //////////////////// Actions \\\\\\\\\\\\\\\\\\\\
     @Step("Navigate to Home Page")
     public HomePage navigate() {
         driver.browser().navigateToURL(url);
         return this;
     }
 
+    @Step("Click on scroll up arrow")
+    public HomePage clickOnScrollUpArrow() {
+        driver.element().click(scrollUpArrow_button);
+        return this;
+    }
     //////////////////// Validations \\\\\\\\\\\\\\\\\\\\
```

Comment:
Please add an empty line here between the method and the Validation section.

Reply:
Done